actions/sync-calendar/main.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/spinnaker/rotation-scheduler/actions/sync-calendar/gcal"
+	"github.com/spinnaker/rotation-scheduler/proto/schedule"
+	"log"
+	"os"
+)
+
+var (
+	schedPath  = flag.String("schedule", "", "Required. Path to schedule YAML file.")
+	jsonKey    = flag.String("jsonKey", "", "Required. The path to the JSON key with access to the Calendar API.")
```

Comment:
Would it make sense to try to use application default credentials for this, which would leave fewer keys floating around?

Reply:
I anticipate this running via GitHub Actions, which being off-GCP, would not have ADC available. 

With that in mind, I started looking at how I would get the key in via Environment variables, and it looks like the [GCP recommended way](https://github.com/GoogleCloudPlatform/github-actions/tree/master/setup-gcloud) base64 encodes the file and the action writes it out to a tmp file. I'll work on this change in a separate PR.